### ISSUE TITLE
Error: Unable to open profiler: Cannot set property 'actionBarContext' of undefined

### ISSUE THREAD

**Issue by LucasWBritz** [microsoft/azuredatastudio#12661]
- Azure Data Studio Version:
Version: 1.22.0 (user setup)
Commit: 77b9a708df3679c26a89ba24a070b3f7a078a510
Date: 2020-09-21T20:23:13.120Z
VS Code: 1.49.0
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:

To be honest I don't know exactly how to reproduce this. But I had 2 tabs open (side by side) at the same time. Then I clicked with the right button on my server and then on 'Launch Profiler'. 

I solved the problem reopening Azure data studio. 
![image](https://user-images.githubusercontent.com/24421665/94587931-a34cdb00-0259-11eb-8033-30645646a9e6.png)

More info: The database I'm connected to is **on Azure**. Not a local database.

**Comment by alanrenmsft** [comment]
yeah, this is usually caused by editors in a bad state, closing all editors(tabs) and restart ADS is the workaround, I will see whether there is something I can do to fix this or at least show more meaningful errors.
